lib/webpacker/manifest.rb

```diff
@@ -46,11 +47,7 @@ def handle_missing_entry(name)
     end
 
     def data
-      if env.development?
-        refresh
-      else
-        @data ||= load
-      end
+      @data ||= load
```

Comment:
This will now only refresh if you're using the compile method here. Not if you're using, say, the watcher.